index.html

```diff
@@ -176,6 +177,44 @@ <h1 class="section-header">About Speedometer 3</h1>
                     </div>
                 </div>
             </section>
+
+            <section id="instructions" data-title="Test Instructions">
+                <a href="#home" class="logo">
+                    <img srcset="resources/logo@2x.png 2x" src="resources/logo.png" alt="Speedometer" />
+                    <div class="version">3.0</div>
+                </a>
+                <div class="section-grid">
+                    <h1 class="section-header">Speedometer 3 Test Instructions</h1>
+                    <div class="section-content">
+                        <h2>General Instructions</h2>
+                        <ol>
+                            <li>Before running Speedometer:</li>
+                            <ul>
+                                <li>Close all other applications.</li>
+                                <li>Close other browser windows and tabs.</li>
+                                <li>Use a fresh browser profile and a single window.</li>
```

Comment:
Maybe we want to explicitly call out that browser extensions should be disabled?